app/models/post.rb

```diff
@@ -1,3 +1,14 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
+  has_many :comments, class_name: 'Comment', foreign_key: 'post_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'post_id'
+  after_create :update_post_count
+
+  def update_post_count
+    author.increment!(:post_counter)
+  end
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- It would be nice to use one of the Rails Active Callbacks - `after_save` to call the #update_post_counter method every time a new post has been saved to the database, then make the update_post_counter method private 👍🏽